packages/oats/src/generate-types.ts

```diff
@@ -16,8 +16,11 @@ interface ImportDefinition {
 }
 
 // bit of a lie here really
-const voidSchema: oas.SchemaObject = { type: 'void' as any };
-const emptyObjectSchema: oas.SchemaObject = { type: 'object', additionalProperties: false };
+const VOID_SCHEMA: oas.SchemaObject = Object.freeze({ type: 'void' as any });
```

Comment:
do we need freezing really? Asking as we are not using freeze anywhere else so its a bit of a surprise to see it here.

Reply:
I felt like for constants it's a nice way to ensure they will stay constant. If none of the functions below mutates these objects, the behavior will be the same (with freeze or without), otherwise, it's a big no no, so an error will be thrown.

Reply:
does node (in our usage) really throw and not fail silently. Testing this in node repl does not seem to throw.

Reply:
alternatively, we can just replace a constant with factory:
```suggestion
const createVoidSchema = (): oas.SchemaObject => ({ type: 'void' as any });
```
?

Reply:
isn't that ts strict not "strict"?

Reply:
well a factory would do but would maybe be simpler to just not care

Reply:
> isn't that ts strict not "strict"?

yes, it's `alwaysStrict` that emits 'use strict' - https://www.typescriptlang.org/tsconfig#alwaysStrict

> but would maybe be simpler to just not care

mm, sure. I felt like it can be a good practice considering that in this case the constant is an object not a primitive. But sure, let's not care until there's a reason to care (seriously).